### PR TITLE
Basic setup

### PR DESCRIPTION
**Pull request by mosaaleb**
- install redux & react-redux packages
- heroku deployment
- requested working alone in this projects

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+linters:
+  eslint:
+    config: './.eslintrc.json'
+files:
+    ignore:
+      - 'tailwind.js'
```
Comment (bolah2009) [inline .stickler.yml]: Setup appropriate code linter. 

Let's use the config file here (https://github.com/microverseinc/linters-config/blob/master/react-redux/.stickler.yml) instead. I see `tailwind.js` is ignored in `.eslintignore` already, you don't need to change the stickler file.

Please let the `./.eslintrc.json` be same content with this: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

Comment (mosaaleb) [reply .stickler.yml]: Stickler still gives errors if I haven't ignored the files in stickler configurations. It doesn't see the eslintignore. Also what is the difference between the eslint configs?

Comment (mosaaleb) [reply .stickler.yml]: Also I don't see a requirement to stick to a specific config if I am not ignoring necessary files.

Comment (bolah2009) [reply .stickler.yml]: > Stickler still gives errors if I haven't ignored the files in stickler configurations.

Really? That's weird. 
Is there a commit that shows this? I couldn't find any commit with error on this PR. 

> Also what is the difference between the eslint configs?

Please, see for yourself. I post the link to the two config.

Comment (mosaaleb) [reply .stickler.yml]: All my previous projects which used stickler gives linter errors if I don't ignore those files. 
> Is there a commit that shows this? I couldn't find any commit with error on this PR.
You don't have to.

> Please, see for yourself. I post the link to the two config.
I know exactly the difference I asked about the essence of changes not the actual changes. 

**Still can't see the screenshots for these changes**

**Comment by mosaaleb** [comment]
@bolah2009 

https://github.com/mosaaleb/redux-bookstore/blob/basic-setup/.eslintrc.json#L28-L31

I experienced an issue with previous project when tried to write `label` tag with `input`

check the issues: [1](https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/302) [2](https://github.com/airbnb/javascript/issues/1885)

Rule:
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md

https://github.com/mosaaleb/redux-bookstore/blob/basic-setup/.eslintrc.json#L26
This is because I prefer: 
```
{
firstName: 'bla', 
lastName: 'bla' 
}
// over
{
firstName: 'bla', 
lastName: 'bla',
}
```

**Pull request merged**